packages/default-theme/__e2e__/integration/checkout_desktop.spec.js

```diff
@@ -0,0 +1,38 @@
+describe('Test Checkout funcionality Desktop view' , () => {
+    
+    beforeEach(() => {
+       cy.viewport(1280,800)
+       // run these tests as if in a desktop
+       // browser with a 800p monitor
+       
+    })
+    describe('Add product to the cart-product is in the cart', () => {
+        it('Navigate to Shopware-pwa site', () => {
+            cy.visit('https://shopware-pwa.storefrontcloud.io/')
+            cy.wait(1000)
+        })
+        it('Get first product from home page product listing', ()=> {
+            cy.get(':nth-child(1) > .sf-product-card__link > .product-card-title > .product-card-title__title')
+                .click({ force: true})
+        })
+        it('Check if card is empty', () => {
+            cy.get('.sf-circle-icon__badge').should('not.exist')
+        })
+        it('Add product to the cart', () => {
+            cy.get('[data-test=sf-add-to-card] > .sf-button').click({ force: true})
```

Comment:
It would be better to use assertion in such a scenario. I can show you how :)

Reply:
@alinadivante I will be grateful for an example matching this scenario :-)